packages/core/tools/_focused.scss

```diff
@@ -11,7 +11,7 @@
 
 @mixin nhsuk-focused-text {
   background-color: $nhsuk-focus-color;
-  box-shadow: 0 -2px $nhsuk-focus-color, 0 $nhsuk-focus-width $nhsuk-focus-text-color;
+  box-shadow: 0 -2px $nhsuk-focus-color, 0 4px $nhsuk-focus-text-color;
```

Comment:
Reverting the `$nhsuk-focus-width` back to `4px` means we can use the variable instead of the `4px`.